packages/support/lib/env.js

```diff
@@ -112,21 +112,23 @@ export const readPackageInDir = _.memoize(
  * - If `APPIUM_HOME` is set in the environment, use that
  * - If we have an `extensions.yaml` in {@linkcode DEFAULT_APPIUM_HOME}, then use that.
  * - If we find a `package.json` in or above `cwd` and {@linkcode shouldUseCwdForAppiumHome} returns `true`, then use the directory containing the `package.json`.
+ *
+ * All returned paths will be absolute.
  */
 export const resolveAppiumHome = _.memoize(
   /**
    * @param {string} [cwd] - Current working directory.  _Must_ be absolute, if specified.
    * @returns {Promise<string>}
    */
   async function _resolveAppiumHome(cwd = process.cwd()) {
-    if (process.env.APPIUM_HOME) {
-      return process.env.APPIUM_HOME;
-    }
-
     if (!path.isAbsolute(cwd)) {
       throw new TypeError('`cwd` parameter must be an absolute path');
     }
 
+    if (process.env.APPIUM_HOME) {
```

Comment:
should we also validate if `process.env.APPIUM_HOME` is a relative path?

Reply:
If it's an absolute path, `path.resolve(someAbsolutePath, process.env.APPIUM_HOME)` will just be `process.env.APPIUM_HOME`.  If it's relative, then `path.resolve()` will make it absolute.  I don't see any need to check if it's relative before performing `path.resolve()`.

Reply:
this was not obvious to me